src/index.ts

```diff
@@ -71,14 +74,14 @@ async function getCodemodPath(pathToCodemod: string, options: TSOptions) {
   return path.resolve(pathToCodemod);
 }
 
-function getProgressUI(logOpts: Pick<Options, 'porcelain' | 'jsonOutput'>, totalCount: number) {
+function getProgressUI(logOpts: Pick<Options, 'porcelain' | 'jsonOutput'>, totalCount: number, codemodName: string) {
   if (logOpts.porcelain || logOpts.jsonOutput) {
     // This is intentional.
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     return {tick() {}};
   }
 
-  return new ProgressBar(':bar (:current/:total, :percent)', {total: totalCount});
+  return new ProgressBar(`[${codemodName}] :bar (:current/:total, :percent)`, {total: totalCount});
```

Comment:
This is a usability improvement when you have multiple phases of codemods. Previously, you'd just see a bunch of progress bars, but not know which codemod a progress bar referred to.